apps/admin/frontend/src/app.test.tsx

```diff
@@ -106,7 +108,6 @@ beforeEach(() => {
 
 afterEach(() => {
   delete window.kiosk;
-  MockDate.reset();
```

Comment:
This `reset` was calling tests to fail because it was triggering session time limits. I removed it because it isn't actually needed - we're already resetting the date in this file's `beforeEach`